src/Jackett.Common/Definitions/divteam.yml

```diff
@@ -116,7 +116,17 @@
         selector: a[href^="download.php?id="]
         attribute: href
       title:
-        selector: a[href^="index.php?page=torrent-details"]
+        selector: a[href^="index.php?page=torrent-details"]:contains("VOSE")
```

Comment:
you are getting this unexpected result because you have failed to take into account that there are some rows that contain a selector that also has a **href** that begins with `index.php?page=torrent-details` and does not contains `VOSE`
there is the title href, and some rows also have a comment href ;-)
```html
<tr>
    <td style="width:3px;"></td>
    <td style="text-align: center;" class="" ><a href="index.php?page=torrents&amp;category=82"><img src="https://divteam.com/style/DivTeam/images/categories/webdl 1080.png" border="0"  alt="Pelis Web-DL 1080" title="Pelis Web-DL 1080" /></a></td>
    <td class="">
        <table>
            <tbody>
                <tr>
                    <td><a href="index.php?page=torrent-details-ocio&id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6"onmouseover=" return overlib('<img src=torrentimg/0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6.png width=200 border=0>', CENTER);" onmouseout="return nd();">A Esmorga 2014 1080p WEB DL VOSE AAC 2 0 H264 TMd</a></td>
                </tr>
                <tr>
                    <td><img title="Free Leech" src="images/freeleech.gif" alt="Free Leech"/></td>
                </tr>
            </tbody>
        </table>
    </td>
    <td class=" ">3.11 GB</td>
    <td style="white-space:wrap;padding-left:10px;" class=""><a href="index.php?page=torrent-details-ocio&id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6#comments" class="btn btn-link">1</a>
    <td style="white-space:wrap; text-align:center;" class="">30/03/2020</td>
    <if:usacotl4>
    <td class=""><a href='index.php?page=userdetails&id=4865'><span style='color:#FB9409'>Jordan</span></a></td>
    </if:usacotl4>
    <td class="#FFFF00" style="text-align: center;"><a href="index.php?page=peers&amp;id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6" title="Click here to view peers details">21</a></td>
    <td class="#FF0000" style="text-align: center;"><a href="index.php?page=peers&amp;id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6" title="Click here to view peers details">0</a></td>
    <td style="text-align: center;" class=""><a href="index.php?page=torrent_history&amp;id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6">38</a></td>
    <td style="text-align: center;" class=""><a href="download.php?id=0c7a0c3a21f4b1c3ddc289053b02db9498fab4c6&amp;f=A+Esmorga+2014+1080p+WEB-DL+VOSE+AAC.2.0+H264+TMd.torrent"><i class='fa fa-download fa-3x' aria-hidden='true'></i></a>
    </td>
</tr>
```

so if you are going to switch to optional titles you will need to exclude the comments href, by specifically selecting the first href to be examined.

```suggestion
        selector: a[href^="index.php?page=torrent-details"][onmouseover]:contains("VOSE")
```